src/components/Header/components/LocationSearch/LocationSearch.module.scss

```diff
@@ -0,0 +1,14 @@
+.currentLocation {
+  .title {
+    color: #aaaaaa;
```

Comment:
I sugggest to never use hardcoded color values. Create colors.scss file with variables

Reply:
Done. I have created style constants.